sites/default/statement.inc.php

```diff
@@ -32,7 +32,7 @@
  *
  *  2.  Branded Statement, whose core is build from 1., the original statement, using HTML2PDF.
  *
- *      To customize 2., add your practice location/images/practice_logo.gif 
```

Comment:
Sorry about the whitespace removal thingy, my editor (Atom) does it automatically. I shall look into disabling that for this project. Or maybe we can create an issue out of this and removal redundant whitespace from every file in a single PR